docs/sources/operators-guide/architecture/components/ruler/index.md

```diff
@@ -75,3 +75,14 @@ The following example shows a local storage definition:
 
 The ruler looks for tenant rules in the `/tmp/rules/<TENANT ID>` directory.
 The ruler requires rule files to be in the [Prometheus format](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules).
+
+## Evaluate rules using query-frontend
+
+Despite including its own built-in querier, the ruler can be configured to delegate evaluation of rule expressions to the query-frontend.
```

Comment:
Your doc edit is great. My only take is that if you start reading this doc from the beginning, it's not clear the ruler supports 2 operational modes. I think we should add a paragraph before "Recording rules", explaining the 2 operational modes. I would move the `ruler.svg` under that paragraph and I would work with UI team to create a chart for the new operational mode too.

Reply:
@pracucci except for the changes in the `ruler.svg` file, I've [included](https://github.com/grafana/mimir/pull/1714/commits/c08cc5fb02e0e58bd38a840048bce60a7b52617e) the new section you suggested, explaining the different operational modes for the ruler. Would be great if you could do the tech review to confirm that everything makes sense.